learning_paths/api/v1/utils.py

```diff
@@ -67,3 +68,49 @@ def get_aggregate_progress(request, learning_path):
 
     aggregate_progress = total_completion / total_courses
     return aggregate_progress
+
+
+def is_learning_path_completed(user, learning_path):
```

Comment:
Maybe something along `is_completion_threshold_met` would be more accurate? We want to clearly distinguish between the completion/progress and grading criteria to avoid confusion.

Reply:
Makes sense. Updated it.